src/commands/registerCommands.ts

```diff
@@ -57,4 +58,5 @@ export function registerCommands(): void {
 
     registerCommand('azureResourceGroups.clearActivities', clearActivities);
     registerCommand('azureResourceGroups.toggleShowAllResources', toggleShowAllResources);
+    registerCommand('azureResourceGroups.showGroupOptions', showGroupOptions);
```

Comment:
It might be best to just include it in the package manifest along with an activation event, and suppress it there from the command palette. If the walkthrough opens and the user very quickly clicks that button _before_ the RG extension has time to activate, it will throw an error.